### ISSUE TITLE
What about making an official augeas docker image

### ISSUE THREAD

**Issue by Salokyn** [hercules-team/augeas#623]
Here is a Dockerfile I wrote:

```Dockerfile
FROM alpine

RUN apk add --no-cache libxml2 readline

RUN set -xe \
  ; apk add --no-cache --virtual=.build g++ make bison flex readline-dev libxml2-dev git automake autoconf libtool pkgconf coreutils \
 && git clone git://github.com/hercules-team/augeas \
 && cd augeas \
 && ./autogen.sh && make && make install \
 && cd - \
  ; rm -fr /augeas \
  ; apk del .build

CMD augtool
```

The image weights 20 MB

**Comment by raphink** [comment]
The `augeas-sandbox` tools could probably be added, too.

**Comment by Salokyn** [comment]
You mean adding aug* scripts from [raphink/augeas-sandbox](https://github.com/raphink/augeas-sandbox) in `/usr/local/bin` ?

I could propose a PR if I knew where to put the Dockerfile in the repo.

**Comment by raphink** [comment]
Right, I'd like @lutter's opinion on this first.

**Comment by lutter** [comment]
I would just create a new `docker/` directory at the toplevel and put the `Dockerfile` and anything else needed for that there. I'd also be totally fine with adding whatever is still relevant from @raphink's augeas-sandbox into the `bin/` directory

**Comment by raphink** [comment]
Fixed in #650
